Kiss.Bff/ZaakGerichtWerken/Contactverzoeken/ReadContactverzoekenVragenSets.cs

```diff
@@ -15,15 +15,15 @@ public ReadContactverzoekenVragenSets(BeheerDbContext db)
         {
             _db = db;
         }
-
+        
         [HttpGet("/api/contactverzoekvragensets")]
         public async Task<IActionResult> Get(CancellationToken token)
         {
             var list = await _db.ContactVerzoekVragenSets.ToListAsync();
 
             if (list == null || !list.Any())
```

Comment:
die check op !list.Any() kan eraf

Reply:
en tip, geen prio: als je AsAsyncEnumerable() gebruikt in plaats van ToListAsync(), hoeft niet de hele lijst in het geheugen geladen te worden, maar wordt deze gestreamd naar de response body.